upload.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# It reads latest uploaded version and compares it with currently built one.
+# If they are different, script will upload new jdk version and update file with latest name.
+set -eo pipefail
+
+echo "Checking $2..."
+latest=$(curl --silent -L http://packages.$2/buildpack-java/latest$1)
+echo "Latest version: $latest"
+
+current=$3
+echo "Current version: $current"
+
+if [ "$latest" == "$current" ]; then
```

Comment:
It might happen, for some broken reason, that latest is older than current this would upload an outdated version to all production buckets. Maybe you can do a comparison like this one https://github.com/cloudControl/buildpack-clojure/blob/master/etc/upload-latest-leiningen.sh#L26